.gitlab-ci.yml

```
@@ -13,7 +13,7 @@ stages:
     - docker-build
   script:
     - docker login -u $CI_REGISTRY_USER -p $CI_REGISTRY_PASSWORD $CI_REGISTRY
-    - ./tools/docker/image-build.sh --image "${CI_JOB_NAME#image-build-}" --tag $CI_PIPELINE_ID --push
+    - ./tools/docker/image-build.sh --image "${CI_JOB_NAME#image-build-}" --tag $CI_PIPELINE_ID --tag latest --push
```

This is not good. We *don't* want to push the latest tag for every pipeline run for any random non-reviewed branch. So this should be done only for master. Since we don't do cppcheck on this, we can code it as:
```suggestion
    - if [ "$CI_COMMIT_BRANCH" = "master" ]; then latest_tag='--tag latest'; fi
    - ./tools/docker/image-build.sh --image "${CI_JOB_NAME#image-build-}" --tag $CI_PIPELINE_ID $latest_tag --push
```